router/pkg/config/config.schema.json

```diff
@@ -130,6 +148,53 @@
         }
       }
     },
+    "automatic_persisted_queries": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The configuration for the automatic persisted queries (APQ).",
+      "required": ["enabled"],
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable fallback storage provider. The default value is false."
+        },
+        "cache": {
```

Comment:
Can you add a description on what the cache is used for vs the storage?
Maybe this description could also live one layer above, but it's also ok to explain it in the cache and storage sections.